src/daemon/daemon.cpp

```diff
@@ -3671,6 +3673,10 @@ void mp::Daemon::run_commands_at_boot_on_instance(const std::string& name, fmt::
 
             warned_exec_failure = true;
         }
+        catch (const ExitlessSSHProcessException&) // Timeout does not mean there is a configuration error.
+        {
+            add_fmt_to(warnings, "Timeout executing command \"{}\" in instance {}.\n", *current_command, name);
```

Comment:
Not sure if this really needs to be communicated back to the client.  I think logging the warning would be better.